src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.tsx

```diff
@@ -136,8 +125,8 @@ type Props = {
   // Callbacks //
   onChangeProjects?: (val: number[]) => void;
   onUpdateProjects?: (selectedProjects: number[]) => void;
-  onChangeEnvironments?: (environments: Environment[]) => void;
-  onUpdateEnvironments?: (environments: Environment[]) => void;
+  onChangeEnvironments?: (environments: string[]) => void;
```

Comment:
environments are no longer an array of objects?

```
export type Environment = {
  id: string;
  displayName: string;
  name: string;

  // XXX: Provided by the backend but unused due to `getUrlRoutingName()`
  // urlRoutingName: string;
};
```

Reply:
Oh they are, just not here. Here we only have the names.